src/components/SortableTable/SortableTable.tsx

```diff
@@ -66,22 +108,62 @@ const SortableTable: React.FC<SortableTableProps> = ({ data, columns }) => {
     return data;
   }, [data, sortState]);
 
+  const TableCellContent = ({ content }: TableCellContentProps) => {
+    const isArrayOfStrings = (content: any): content is HtmlString[] => {
+      return (
+        Array.isArray(content) &&
+        content.every((item) => typeof item === "string")
+      );
+    };
+    if (typeof content === "object" && content !== null) {
+      const { mean, std } = content as Stats;
+      if (mean !== undefined && std !== undefined) {
+        return (
+          <>
+            {mean} ± {std}
+          </>
+        );
+      }
+    }
+    if (isArrayOfStrings(content)) {
+      const renderedContent = content.map((item, index) => (
+        <Fragment key={index}>
+          {index > 0 && ", "} {/* Add a comma after each element */}
+          <HTMLString html={item} />
+        </Fragment>
+      ));
+      return <>{renderedContent}</>;
+    }
+    return <>{content}</>;
+  };
+
+  // TODO: use flex box in css
   return (
     <table>
       <thead>
-        <tr>
-          {columns.map((column) => (
-            <th key={column.key} onClick={() => handleSort(column.key)}>
-              {column.header} {getSortIcon(column.key)}
+        {horizontalHeaders.map((header) => (
+          <tr>
+            <th onClick={() => handleSort(header.key)}>
+              {header.value}
+              <span className="sort-icon">{getSortIcon(header.key)}</span>
```

Comment:
Rendering all the sort icons looks a bit busy.
Would be cool to render icon of key that is not active as gray or hidden and black on hover.

Reply:
Fixed!